rasa/nlu/components.py

```diff
@@ -229,19 +226,17 @@ def validate_required_components_from_data(
         )
 
     if data.entity_examples and not any_components_in_pipeline(
-        {"DIETClassifier"}, pipeline
+        {"DIETClassifier", "CRFEntityExtractor"}, pipeline
```

Comment:
you mention in the PR that this contains changes for `DIETClassifier` as well - are you still planning to add that to this PR?

Reply:
changes are already done